docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -674,6 +674,8 @@ export default ({ children }) => (
 )
 ```
 
+To restore the default from Gatsby v1, see the [plugin docs](/packages/gatsby-plugin-sass/README.md#how-to-use).
```

Comment:
In this section there is talk about css modules (not sass specific version of it) - we should make it clear. Like if you are using less - see less plugin docs, if sass - see sass plugins docs. I will prepare code snippet for restoring v1 default for vanilla css modules

Reply:
Good call—updated (will need to tweak the doc further when you make it possible with vanilla).

Reply:
So here's snippet that can be used for vanilla css modules in gatsby-node.js file that restores v1 behaviour:

```js
const cssLoaderRe = /\/css-loader\//
const targetFile = `.module.css`

const processRule = rule => {
  if (rule.oneOf) {
    return {
      ...rule,
      oneOf: rule.oneOf.map(processRule),
    }
  }

  if (!rule.test.test(targetFile)) {
    return rule
  }

  if (Array.isArray(rule.use)) {
    return {
      ...rule,
      use: rule.use.map(use => {
        if (!cssLoaderRe.test(use.loader)) {
          return use
        }

        // adjust css-loader options
        return {
          ...use,
          options: {
            ...use.options,
            camelCase: false,
          },
        }
      }),
    }
  }

  return rule
}

exports.onCreateWebpackConfig = ({ getConfig, actions }) => {
  const config = getConfig()

  const newConfig = {
    ...config,
    module: {
      ...config.module,
      rules: config.module.rules.map(processRule),
    },
  }
  actions.replaceWebpackConfig(newConfig)
}
```